src/pages/workspace/expensifyCard/WorkspaceCardSettingsPage.tsx

```diff
@@ -66,7 +64,7 @@ function WorkspaceCardSettingsPage({route}: WorkspaceCardSettingsPageProps) {
                                 description={translate('workspace.expensifyCard.settlementFrequency')}
                                 title={translate(`workspace.expensifyCard.frequency.${settlementFrequency}`)}
                                 shouldShowRightIcon={settlementFrequency !== CONST.EXPENSIFY_CARD.FREQUENCY_SETTING.DAILY}
-                                // interactive={!isSettlementFrequencyBlocked}
+                                interactive={!isSettlementFrequencyBlocked}
```

Comment:
NAB: Maybe add  `disabled` props


```suggestion
                                interactive={!isSettlementFrequencyBlocked}
                                disabled={isSettlementFrequencyBlocked}

```

Reply:
@DylanDylann It adds the opacity, I'm not sure that it's the expected behaviour

![image](https://github.com/user-attachments/assets/c5777cec-02de-4766-aa6c-480f89413a78)

Reply:
Yeah, It's only my suggestion and also be a minor thing

Reply:
I don't think the disabled prop is necessary in this case